spyder_kernels/utils/pythonenv.py

```diff
@@ -61,6 +62,33 @@ def get_conda_env_path(pyexec, quote=False):
     return conda_env
 
 
+def get_pixi_manifest_path_and_env_name(pyexec, quote=False):
+    pyexec_path = Path(pyexec.replace("\\", "/"))
+    pixi_env_path = pyexec_path.parent
+    pixi_env_name = pixi_env_path.name
+    pixi_dir_path = pixi_env_path.parent.parent
```

Comment:
```suggestion
    pixi_dir_path = pixi_env_path.parents[1]
```

This is a simpler way to access the parent's parent.